### ISSUE TITLE
Add italian 2-letter codes for provinces

### ISSUE THREAD

**Issue by mircobabini** [country-regions/country-region-data#78]
As shown here: https://github.com/woocommerce/woocommerce/blob/10f92ca09101cb119f1fe9ed37b4967b2940e17e/i18n/states.php#L703-L810

**Comment by Martijn097** [comment]
True, most shipping carriers expect the province and not the region. We currently override Italy for this
